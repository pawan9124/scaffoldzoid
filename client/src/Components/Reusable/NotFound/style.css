.notfound_section {
  display: flex;
  justify-content: center;
}
.notfound_heading {
  color: red;
  font-style: italic;
  font-family: "Courier New", Courier, monospace;
}
