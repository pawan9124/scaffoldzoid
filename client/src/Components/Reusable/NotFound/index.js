import React from "react";
import "./style.css";

const NotFound = () => {
  return (
    <div className="notfound_section" data-test="NotFoundComponent">
      <h1 className="notfound_heading">Not found</h1>
    </div>
  );
};

export default NotFound;
