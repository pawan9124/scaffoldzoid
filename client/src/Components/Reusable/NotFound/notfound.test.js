import React from "react";
import { shallow } from "enzyme";
import { findByTestAttr, checkProps } from "../../../testUtils";
import NotFound from "./index";
import renderer from "react-test-renderer";

describe("Taking snap shot", () => {
  it("Taking the snapshot of the Component", () => {
    const tree = renderer.create(<NotFound />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe("Checking the render of component", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<NotFound />);
  });
  it("Should Render a component", () => {
    const button = findByTestAttr(wrapper, "NotFoundComponent");
    expect(button.length).toBe(1);
  });
});
