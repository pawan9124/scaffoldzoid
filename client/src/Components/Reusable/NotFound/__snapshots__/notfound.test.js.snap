// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Taking snap shot Taking the snapshot of the Component 1`] = `
<div
  className="notfound_section"
  data-test="NotFoundComponent"
>
  <h1
    className="notfound_heading"
  >
    Not found
  </h1>
</div>
`;
